Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/NolPayTokenizationViewModel.swift

```diff
@@ -0,0 +1,257 @@
+//
+//  NolPayTokenizationViewModel.swift
+//  PrimerSDK
+//
+//  Created by Boris on 28.8.23..
+//
+#if canImport(UIKit)
+
+import Foundation
+import SafariServices
+import UIKit
+
+class NolPayTokenizationViewModel: PaymentMethodTokenizationViewModel {
+    
+    private var redirectUrl: URL!
```

Comment:
Are all of these implicitly unwrapped optionals necessary?

For example, `URL(string:)` can fail, I think we risk crashing the app if we end up setting this property with bad data.

Small example, will crash in playground:
```
class TestURL {
    var redirectURL: URL!
    
    func validate(url: String) {
        redirectURL = URL(string: url)
        print(redirectURL.isFileURL) //try use it in any way
    }
}

TestURL().validate(url: "")
```

I see you are assigning them inside `if let`s, which might make it safe - it just feels like a smell. Happy to discuss